public/js/graph/react_node.js.jsx

```diff
@@ -0,0 +1,113 @@
+function renderReactGraph() {
+    React.render(
+        <ReactSVG width={1195} height={650}/>,
+        document.getElementById('react-graph')
+    );
+}
+
+function getAttributes(node_named_map) {
+    var attrs = [];
+    //Traversing a NodeNamedMap type
+    Array.prototype.slice.call(node_named_map).forEach(function(item) {
+        attrs[item.name] = item.value;
+    });
+    return attrs;
+}
+
+function getStyles(styles_strings) {
+    if (!styles_strings){
+        return {};
+    }
+    var styles = {};
+        styles_strings.split(";").map(function(key, value){
```

Comment:
The indentation here appears to be off, and there's a space missing between here.